blackjax/inference/trajectory.py

```diff
@@ -370,7 +564,7 @@ def expand_once(expansion_state):
                 new_trajectory,
                 termination_state,
                 is_diverging,
-                has_terminated,
+                is_turning_subtree,
                 num_steps,
```

Comment:
`num_steps` is no longer necessary. I am trying to find a solution for all this state passing that is becoming quite ugly.

Reply:
Yeah it is in fact redundant with the `num_states` in `Trajectory`